diff --git a/node_modules/@tcgdex/sdk/dist/tcgdex.node.d.mts b/node_modules/@tcgdex/sdk/dist/tcgdex.node.d.mts
index dc246ff..a87d594 100644
--- a/node_modules/@tcgdex/sdk/dist/tcgdex.node.d.mts
+++ b/node_modules/@tcgdex/sdk/dist/tcgdex.node.d.mts
@@ -534,6 +534,11 @@ declare class Set extends Model {
     };
     cards: Array<CardResume>;
     boosters?: Array<Booster>;
+
+    abbreviation?: {    
+        
+        official?: string;
+    };
     getSerie(): Promise<Serie | null>;
     protected fill(obj: object): void;
 }
